torch/_inductor/codegen/common.py

```diff
@@ -349,6 +349,10 @@ def _print_ModularIndexing(self, expr):
 
     def _print_FloorDiv(self, expr):
         x, div = expr.args
+        if div == -1:
+            return f"{self.paren(self.doprint(x * -1))}"
+        elif div == 1:
+            return f"{self.paren(self.doprint(x))}"
```

Comment:
Is this condition already covered here?
https://github.com/pytorch/pytorch/blob/2c3ab605068b7efc09f63c62048671750f69d6da/torch/utils/_sympy/functions.py#L65-L68

Reply:
Yes, it is. I've updated the code to remove the duplicated condition.